### DISCUSSION TITLE
What to do in case of same product with different prices for the same city ?

### DISCUSSION THREAD

**Discussion by abdelmajidhafidi** [geeksblabla/blanat#28] [Q&A] [answered]
Salam, 
first of all, I must congratulate you on BlaBlaConf and I wish you all the best for next days and next editions.

I have a question about this challenge, in the description, it is mentioned that "Each city may have the same product duplicated several times with different or identical prices.".

if we find a case like this, do we have to add the amount of the product to the city's total (even though we already found it with another price and we already added it) or we have to choose only the cheapest price of the product?

**Comment by aboullaite** [comment, marked as answer]
For the city total price it needs to be the sum of all, even duplicate products. I hope that answers it

**Reply by abdelmajidhafidi** [reply]
Yes, thank you !!
